help-beta/astro.config.mjs

```diff
@@ -64,6 +64,7 @@ export default defineConfig({
             title: "Zulip help center",
             components: {
                 Footer: "./src/components/Footer.astro",
+                Head: "./src/components/Head.astro",
```

Comment:
Starlight's recommendation to override the head component: https://starlight.astro.build/reference/configuration/#head